packages/react-ui/internal/themes/FlatTheme.ts

```diff
@@ -191,7 +191,7 @@ export class FlatTheme extends (class {} as typeof DefaultThemeInternal) {
   public static radioSize = '18px';
   public static radioSizeAfter = '20px';
   public static radioVerticalAlign = 'top';
-  public static radioBgImage = 'none';
+  public static radioBgImage = '#fff';
```

Comment:
Это не валидное значение для `background-image`. Из-за чего тебе и пришлось добавить новое правило в `Radio.style.ts`.

Вот валидное значение:
```suggestion
  public static radioBgImage = 'linear-gradient(#fff,#fff)';
```